protocol/x/delaymsg/keeper/dispatch.go

```diff
@@ -29,13 +29,13 @@ func DispatchMessagesForBlock(k types.DelayMsgKeeper, ctx sdk.Context) {
 	for _, id := range blockMessageIds.Ids {
 		delayedMsg, found := k.GetMessage(ctx, id)
 		if !found {
-			k.Logger(ctx).Error("delayed message %v not found", id)
+			k.Logger(ctx).Error("delayed message not found", "id", id)
 			continue
 		}
 
 		msg, err := delayedMsg.GetMessage()
 		if err != nil {
-			k.Logger(ctx).Error("failed to decode delayed message with id %v: %v", id, err)
+			k.Logger(ctx).Error("failed to decode delayed message", "id", id, "error", err)
```

Comment:
Nit: there is an `ErrorLogKey` constant that can be used here instead of "error". If you could add one for `id` as part of this PR too, that would be great! But not required.